.github/workflows/dart.yml

```diff
@@ -60,8 +60,8 @@ jobs:
       - uses: actions/checkout@v2
       - run: |
           pub get
-          dartanalyzer --fatal-warnings ./
-          dartfmt -n --set-exit-if-changed ./
+          dartanalyzer ./
```

Comment:
`--fatal-warnings` is on by default
`--set-exit-if-changed` does not exist for dartfmt